tasks/package.js

```diff
@@ -50,14 +51,18 @@ module.exports = function(grunt) {
     tasks.push('clean:packages');
     tasks.push('copy:package');
 
-    // Rebuild composer dependencies without dev.
-    grunt.config(['composer'], {
-      options: {
-        flags: ['no-dev'],
-        cwd: destPath
-      }
-    });
-    tasks.push('composer:install');
+    // If the `composer.json` file is being packaged, rebuild composer dependencies without dev.
+    if (projFiles.find(function (pattern) {
+      return pattern.startsWith('composer');
```

Comment:
It shouldn't be recursive, it's just looking at the actual `projFiles` array specification (`composer.*` is the pattern in the test config).